bot/app/__init__.py

```diff
@@ -26,59 +26,95 @@
 
 # load env variables from .env
 load_dotenv()
+API_TOKEN = os.getenv('API_TOKEN')
 
 app = Flask(__name__)
-
-API_TOKEN = os.getenv('API_TOKEN')
 bot = telebot.TeleBot(API_TOKEN)
 
-img_dir = os.path.abspath('../bot/img')
+img_dir = os.path.abspath('../bot/app/img')
 
 @bot.message_handler(commands=['start', 'help'])
 def start(message):
 
+	"""start and help message handler function
+
+		upon sending /start or /help all available commands are returned. 
+		in the background the user's id is checked against the db and added,
+		if there is no entry for the user yet
+	"""
```

Comment:
This is not proper google docstring format:

```python
"""Display available commands and register new users.

Upon sending /start or /help all available commands are returned. 
in the background the user's id is checked against the db and added,
if there is no entry for the user yet

Args:
  message: The telebot message object.
"""
```

Reply:
fixed